README.md

```diff
@@ -82,19 +82,28 @@ Before merging a PR:
 
 ## Code Formatting
 
-We use prettier with the default configuration mentionned in the [Solidity Prettier Plugin](https://github.com/prettier-solidity/prettier-plugin-solidity).
-We recommend developers using VS Code to set their local config as below:
+Configure your VSCode to automatically format a file on save, using `forge fmt`:
 
-```
+- Install [emeraldwalk.runonsave](https://marketplace.visualstudio.com/items?itemName=emeraldwalk.RunOnSave)
```

Comment:
I don't think that this should mandatorily in the README. Everyone is free to format on save or not. We can just say that this repo uses forge fmt.

Reply:
I add it because it was in Morpho-Aave-V3 readMe. I don't care removing it from both. What do you think ?

Reply:
I would say yes

Reply:
Are we sure it's formatting on pre-commit?

Reply:
> Are we sure it's formatting on pre-commit?

We can't, because it relies on `husky` which is installed if the dev wants to
But we could setup a CI to check

Reply:
I'm still in favor of removing these details from the readme